src/app/rundown-view/components/rundown/rundown.component.spec.ts

```diff
@@ -10,11 +10,66 @@ import { Logger } from '../../../core/abstractions/logger.service'
 import { RundownEventObserver } from 'src/app/core/services/rundown-event-observer.service'
 import { ActionStateService } from '../../../shared/services/action-state.service'
 import { EntityParser } from '../../../core/abstractions/entity-parser.service'
+import { MiniShelfStateService } from '../../services/mini-shelf-state.service'
+import { Observable, Subscription } from 'rxjs'
+import { TestEntityFactory } from '../../../test/factories/test-entity.factory'
+import { RundownTimingContext } from '../../../core/models/rundown-timing-context'
+import { Rundown } from '../../../core/models/rundown'
+import { Action } from '../../../shared/models/action'
+import { SimpleChange, SimpleChanges } from '@angular/core'
+import { TestLoggerFactory } from '../../../test/factories/test-logger.factory'
 
+let fixture: ComponentFixture<RundownComponent>
 describe('RundownComponent', () => {
-  it('should create', async () => {
-    const component = await configureTestBed()
+  it('should create and receive rundown, then updates MiniShelves and subscribes several times', async () => {
+    const mockedMiniShelfStateService: MiniShelfStateService = createMockOfMiniShelfStateService()
+    const mockedRundownTimingContextStateService: RundownTimingContextStateService = createMockOfRundownTimingContextStateService()
+    const mockedActionStateService: ActionStateService = createMockOfActionStateService()
+    const mockedRundownEventObserver: RundownEventObserver = createMockOfRundownEventObserver()
+
+    const mockedRundown: Rundown = new TestEntityFactory().createRundown()
+    const component: RundownComponent = await configureTestBed({
+      mockedRundown: mockedRundown,
+      mockedRundownEventObserver: mockedRundownEventObserver,
+      mockedMiniShelfStateService: mockedMiniShelfStateService,
+      mockedRundownTimingContextStateService: mockedRundownTimingContextStateService,
+      mockedActionStateService: mockedActionStateService,
+    })
+
     expect(component).toBeTruthy()
+    expect(component.rundown).toEqual(mockedRundown)
+    verify(mockedMiniShelfStateService.updateMiniShelves(mockedRundown)).once()
+
+    verify(mockedRundownTimingContextStateService.subscribeToRundownTimingContext(mockedRundown.id)).once()
+    verify(mockedActionStateService.subscribeToRundownActions(mockedRundown.id)).once()
+
+    verify(mockedRundownEventObserver.subscribeToRundownDeactivation(anything())).once()
+    verify(mockedRundownEventObserver.subscribeToRundownAutoNext(anything())).once()
+    verify(mockedRundownEventObserver.subscribeToRundownSetNext(anything())).once()
+    verify(mockedRundownEventObserver.subscribeToRundownReset(anything())).once()
+    verify(mockedRundownEventObserver.subscribeToRundownUpdates(anything())).once()
+  })
+  it('should update MiniShelves and ActionMappings upon change in rundown', async () => {
```

Comment:
This test is also testing for two different responsibilities.

Sidenote: It would be nice if you had a newline between every test :)